src/QMCWaveFunctions/CMakeLists.txt

```diff
@@ -164,6 +166,32 @@ target_include_directories(qmcwfs PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}")
 target_link_libraries(qmcwfs PUBLIC qmcutil qmcparticle platform_runtime)
 target_link_libraries(qmcwfs PRIVATE einspline platform_LA Math::FFTW3)
 
+# Workaround for broken OMP Target debug in clang as of 5d2ce7663b10c107328a4ae0c678165209e64619
+# but clang 15 main is completely broken for our omptarget code,
+#
+# In the meantime we need to make a new target for just the omp target tainted files
+# so we don't have broken debug for all of QMCWavefunctions
```

Comment:
Remove `workaround... ` which will be at the cmake option.
Here say `# Create qmcwfs_omptarget with source files using OpenMP target offload`
Better with neutral words.

Reply:
I will add neutrality.